flow/Platform.actor.cpp

```diff
@@ -2945,6 +2945,35 @@ std::string readFileBytes(std::string const& filename, int maxSize) {
 	return s;
 }
 
+size_t readFileBytes(std::string const& filename, uint8_t* buff, size_t len) {
```

Comment:
just above this there is a `readFileBytes` function. Few comments on this:

1. I think having the same function twice is bad (even if they have a different interface). Instead I think one should use the other.
2. If you rewrite this and you use blocking IO, I think it would be better to use standard C++ stuff (specifically `fstream`).

Reply:
Good point. I will update the method as recommended. 
Thanks